### PR TITLE
added intel 19.0.3 packages

### PR DESCRIPTION
**Pull request by lee218llnl**

### CODE REVIEW

File: var/spack/repos/builtin/packages/intel-parallel-studio/package.py [hunk 1]
```diff
@@ -81,6 +83,7 @@ class IntelParallelStudio(IntelPackage):
     version('professional.2015.1', '542b78c86beff9d7b01076a7be9c6ddc', url='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/4992/parallel_studio_xe_2015_update1.tgz')
 
     # Composer Edition (basic tier; excluded: MPI/..., Advisor/Inspector/Vtune)
+    version('composer.2019.3',    '15373ac6df2a84e6dd9fa0eac8b5f07ab00cdbb67f494161fd0d4df7a71aff8e', url='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/15272/parallel_studio_xe_2019_update3_composer_edition.tgz')
```
Comment (davydden) [inline var/spack/repos/builtin/packages/intel-parallel-studio/package.py]: looks like there is one white space less here.

Comment (lee218llnl) [reply var/spack/repos/builtin/packages/intel-parallel-studio/package.py]: good eye, I added an extra space.

**Comment by lee218llnl** [comment]
@davydden, does this look good to you now?

**Comment by davydden** [comment]
> does this look good to you now?

yes, thanks for fixing the whitespace

**Pull request merged**